docs/tables/slack_conversation_members.md

```diff
@@ -0,0 +1,30 @@
+# Table: slack_conversation_members
+
+Retrieve members of a conversation.
+
+## Examples
+
+### Get member IDs in the #general channel
+
+```sql
+select
+  channel,
+  id
+from
+  slack_conversation_members
+where
+  channel IN (select id from slack_conversation where is_general);
```

Comment:
```suggestion
  channel in (select id from slack_conversation where is_general);
```

Our keywords are in lowercase, for more info on query formatting please see https://steampipe.io/docs/develop/table-docs-standards#style-conventions-1